styles.css

```diff
@@ -164,6 +164,14 @@ body:not(:has(.ddoc)) {
   }
 }
 
+.sidebar__current-item {
+  @apply text-blue-500 relative;
+}
+
+.sidebar__current-item::before {
+  @apply content-[""] absolute -left-[0.85em] w-[0.5em] text-center h-[0.7em] top-[0.25em] [clip-path:polygon(0%_0%,100%_50%,0%_100%)] bg-blue-500;
```

Comment:
I think given the secondary indicator here we should bump up the padding on the `ul` to pl-4 instead of pl-2 and then make this `-left-[1em]` to visually center it in the new whitespace otherwise it felt a bit crowded to me.

Reply:
<img width="400" alt="Screenshot 2024-08-09 at 2 48 17 PM" src="https://github.com/user-attachments/assets/82aabeef-70b6-4df2-ac2b-6859ee584899">

Reply:
For posterity: we decided on a different direction for the "current" indicator arrow